app/src/main/java/de/triology/cas/pm/CesSendPasswordResetInstructionsAction.java

```diff
@@ -52,8 +57,20 @@ protected Event doExecute(final RequestContext requestContext) throws Exception
             return getErrorEvent("username.required", "No username is provided", requestContext);
         }
 
-        val email = passwordManagementService.findEmail(query);
-        val phone = passwordManagementService.findPhone(query);
+        String email = "";
+        try {
+            email = passwordManagementService.findEmail(query);
+        } catch (Throwable e) {
+            throw new RuntimeException(e);
+        }
+
+        String phone = "";
+        try {
+            phone = passwordManagementService.findPhone(query);
+        } catch (Throwable e) {
+            throw new RuntimeException(e);
```

Comment:
Do we really want to throw an unchecked exception - crashing because of a missing phone number seems to be a bit hard.